pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java

```diff
@@ -652,7 +652,7 @@ public void splitNamespaceBundle(
             @PathParam("bundle") String bundleRange,
             @QueryParam("authoritative") @DefaultValue("false") boolean authoritative,
             @QueryParam("unload") @DefaultValue("false") boolean unload,
-            @QueryParam("splitBoundaries") @DefaultValue("") List<Long> splitBoundaries) {
+            @ApiParam("splitBoundaries") List<Long> splitBoundaries) {
```

Comment:
Looks like no test coverage here, @nodece Could you please add a unit test for V1 API?

Reply:
Thanks, added.